app/.eslintrc.json

```diff
@@ -1,3 +1,12 @@
 {
-  "extends": "next/core-web-vitals"
+  "plugins": ["@typescript-eslint"],
+  "extends": [
+    "nava",
+    "plugin:@typescript-eslint/recommended",
+    "prettier" 
```

Comment:
Why not also [prettier's recommended config](https://github.com/prettier/eslint-plugin-prettier#recommended-configuration)?